src/node/channel_merger.rs

```diff
@@ -4,11 +4,31 @@ use crate::context::{AudioContextRegistration, BaseAudioContext};
 use crate::render::{
     AudioParamValues, AudioProcessor, AudioRenderQuantum, AudioWorkletGlobalScope,
 };
+use crate::MAX_CHANNELS;
 
 use super::{
     AudioNode, ChannelConfig, ChannelConfigOptions, ChannelCountMode, ChannelInterpretation,
 };
 
+/// Assert that the given number of channels is valid for a ChannelMergerNode
+///
+/// # Panics
+///
+/// This function will panic if:
+/// - the given number of channels is outside the [1, 32] range,
+/// 32 being defined by the MAX_CHANNELS constant.
+///
+#[track_caller]
+#[inline(always)]
+pub(crate) fn assert_valid_number_of_channels(number_of_channels: usize) {
```

Comment:
This one is already defined in `lib.rs` but with `NotSupportedError` instead of `IndexSizeError`, weird